Base: Add new Mouse + MouseSettings help (#15055)
SerenityOS/serenity: Base/usr/share/man/man1/MouseSettings.md

## Name

> ![Icon](/res/icons/16x16/app-mouse.png) MouseSettings - MouseSettings application

Reviewer: Application names should still be stylized with spaces :)

```suggestion
![Icon](/res/icons/16x16/app-mouse.png) Mouse Settings - Mouse settings application
```

[Open](file:///bin/MouseSettings)

## Synopsis

```**sh
$ MouseSettings
```

## Description

> `MouseSettings` is an application that displays advance mouse properties for user configuration.

Reviewer: ```suggestion
Mouse Settings is an application that displays advanced mouse properties for user configuration.
```

> 
> Under Mouse tab, user can set mouse cursor speed, scroll wheel step size increments, double click speed and button configuration to switch primary and secondary buttons (for left-handed users).

Reviewer: ```suggestion
Under the _Mouse_ tab, the user can set mouse cursor speed, scroll wheel step size increments, double click speed and button configuration to switch primary and secondary buttons (for left-handed users).
```

> 
> Cursor Theme tab provides the user with drop-down list of available default cursor themes, Default and Dark.

Reviewer: ```suggestion
The _Cursor Theme_ tab provides the user with a drop-down list of available default cursor themes.
```

Reviewer: Let's not assume which cursor themes are installed)

Reviewer: Actually it's probably fine, updated the suggestion.

> 
> Supplemental cursors are available under Ports/serenity-theming app. Some supplemental cursor themes are Durrque, Chillychilly, Jakande, Vanliga and Vanliga-Dark among other things.

Reviewer: ```suggestion
Supplemental cursors are available via the `serenity-theming` port. Some supplemental cursor themes include Durrque, Chillychilly, Jakande, Vanliga, and Vanliga-Dark, among others.
```

> 
> Cursor Highlight tab allows the user to change highlight color, opacity and size. The toggle key to enable and disable cursor highlight is Super+H.
> 

Reviewer: ```suggestion
The _Cursor Highlight_ tab allows the user to change highlight color, opacity and size. The global keyboard shortcut to enable and disable cursor highlighting is Super+H.
```

> Mouse Settings is an application that displays advanced mouse properties for user configuration.
> 
> 
> Under the _Mouse_ tab, the user can set mouse cursor speed, scroll wheel step size increments, double click speed and button configuration to switch primary and secondary buttons (for left-handed users).
> 
> 
> The _Cursor Theme_ tab provides the user with drop-down list of available default cursor themes, Default and Dark.
> 
> 
> Supplemental cursors are available via the `serenity-theming` port. Some supplemental cursor themes are Durrque, Chillychilly, Jakande, Vanliga and Vanliga-Dark, among others.
> 
> 
> The _Cursor Highlight_ tab allows the user to change highlight color, opacity and size. The global keyboard shortcut to enable and disable cursor highlighting is Super+H.
> 

Reviewer: If you could remove the double newlines here that would be great :)

```suggestion
Mouse Settings is an application that displays advanced mouse properties for user configuration.

Under the _Mouse_ tab, the user can set mouse cursor speed, scroll wheel step size increments, double click speed and button configuration to switch primary and secondary buttons (for left-handed users).

The _Cursor Theme_ tab provides the user with drop-down list of available default cursor themes, Default and Dark.

Supplemental cursors are available via the `serenity-theming` port. Some supplemental cursor themes are Durrque, Chillychilly, Jakande, Vanliga and Vanliga-Dark, among others.

The _Cursor Highlight_ tab allows the user to change highlight color, opacity and size. The global keyboard shortcut to enable and disable cursor highlighting is Super+H.
```
